packages/blockchain-extension/tutorials/developer-tutorials/createNewIdentity.md

```diff
@@ -6,7 +6,7 @@
 
 * Create a new identity using Local Fabric CA
 
-> Note: This tutorial assumes you have a basic understanding of the VSCode extension and have followed the <a href='../ibm-blockchain-platform-vscode-smart-contract/local-dev.md'> first tutorial</a>.
+> Note: This tutorial assumes you have a basic understanding of the VSCode extension and have followed the <a href='../new-tutorials/basic-tutorials/a1.md'> Basic Tutorials</a> series.
 
 In the very first tutorial (Tutorial 1: Local smart contract development), we used a pre-set identity (`admin`) to connect to the 1 Org Local Fabric gateway and submit/evaluate transactions. In this tutorial, we will learn how to create _our own_ identity.
```

Comment:
Can you change `In the very first tutorial (Tutorial 1: Local smart contract development)` to `In 'Tutorial A4: Invoking a smart contract from VS Code'`.